titles/configuring/master.adoc

```diff
@@ -0,0 +1,25 @@
+include::artifacts/attributes.adoc[]
+:title: Configuring
+:subtitle: Configuring {product}
+:abstract: Learn how to configure {product} for production to work in your IT ecosystem by adding custom config maps and secrets.
+:context: configuring-and-operating
+[id="{context}"]
+= {title}
+
+{abstract}
+
+
+include::assemblies/assembly-provisioning-a-custom-configuration.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-configuring-external-postgresql-databases.adoc[leveloffset=+1]
+
+
+include::modules/configuring-deployment/proc-configuring-deployment-by-using-the-operator.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-configuring-a-proxy.adoc[leveloffset=+1]
+
+
+include::modules/installation/proc-configuring-an-rhdh-instance-with-tls-in-kubernetes.adoc[leveloffset=+1]
+
```

Comment:
The following two modules from assembly-configuring-rhdh-plugins.adoc should be here as well instead of there.
```


// Dynamic plugins cache
include::../modules/dynamic-plugins/con-dynamic-plugins-cache.adoc[ leveloffset=+1]

// Redis cache
include::../modules/dynamic-plugins/proc-installing-and-configuring-redis-cache.adoc[leveloffset=+1]
```

Reply:
```suggestion

include::../modules/dynamic-plugins/con-dynamic-plugins-cache.adoc[ leveloffset=+1]

include::../modules/dynamic-plugins/proc-installing-and-configuring-redis-cache.adoc[leveloffset=+1]

```